packages/calcite-components/src/components/sort-handle/sort-handle.tsx

```diff
@@ -254,10 +254,12 @@ export class SortHandle extends LitElement implements LoadableComponent, Interac
       setPosition,
       setSize,
       widthScale,
+      moveToItems,
     } = this;
     const text = this.getLabel();
 
-    const isDisabled = disabled || !setPosition || !setSize;
+    const isDisabled =
```

Comment:
Is the optional chaining operator on `moveToItems` needed? It looks like it's guaranteed now, but I could be missing something.

Reply:
Its not but I added for extra safety. I can remove it.